vector/src/main/java/im/vector/app/features/media/ImageContentRenderer.kt

```diff
@@ -152,7 +153,11 @@ class ImageContentRenderer @Inject constructor(private val activeSessionHolder:
                     .load(resolvedUrl)
         }
 
-        req.override(Target.SIZE_ORIGINAL, Target.SIZE_ORIGINAL)
+        val maximumScale = (contextView as? PhotoView)?.maximumScale?.toInt() ?: 1
```

Comment:
the default max scale is 3 https://github.com/Baseflow/PhotoView/blob/565505d5cb84f5977771b5d2ccb7726338e77224/photoview/src/main/java/com/github/chrisbanes/photoview/PhotoViewAttacher.java#L42